catalogworks/catalog-contracts
# build: ignore `data/deployments/localhost` directory

### Description

Closes https://github.com/catalogworks/catalog-contracts/issues/22

### How to test

- Clone the repository and change directory into it
- Run `yarn dev`
- Observe that files generated or modified in `data/deployments/localhost/` aren't tracked by Git
